doc/changes/changes_3.0.0.md

```diff
@@ -1,14 +1,20 @@
-# Cloud Storage Extension 2.6.2, released 2023-??-??
+# Cloud Storage Extension 3.0.0, released 2023-??-??
```

Comment:
Why do you increment the major version? Please document breaking changes if there are any.

Reply:
Ahh, good find, I think this was because of context switch from glue-connector